examples/src/main/scala/com/gu/tableversions/examples/TableLoader.scala

```diff
@@ -43,15 +43,4 @@ class TableLoader[T <: Product: TypeTag](
     logger.info(s"Applied the the following changes to sync the Metastore:\n$metastoreChanges")
   }
 
-  def checkout(id: TableVersions.CommitId): Unit = {
-    val checkout = for {
-      _ <- tableVersions.setCurrentVersion(table.name, id)
-      newVersion <- tableVersions.currentVersion(table.name)
-      currentMetastoreVersion <- metastore.currentVersion(table.name)
-      changes = metastore.computeChanges(currentMetastoreVersion, newVersion)
-      _ <- metastore.update(table.name, changes)
-    } yield ()
-
-    checkout.unsafeRunSync()
-  }
```

Comment:
This has moved into the `VersionedMetastore` class.

The remaining `TableLoader` class is now so anemic that it might be worth removing it - just using the versioning API directly is pretty much as concise. I think that's a good sign!